src/common/components/Table/TableHeaderRow.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+import React from 'react';
+import Flex from '../Flex/Flex';
+
+import styles from './styles/headerRow.scss';
+
+import ITableHeaderRowProps from './types/TableHeaderRow.d';
+
+const TableHeaderRow: React.FC<ITableHeaderRowProps> = ({children, className, ...ownProps}) => (
+	<Flex {...ownProps} className={classNames(styles.header, {[className as string]: !!className})} >
```

Comment:
as I believe, this check is not need. If className is undefined, class will be ignored